Extensions/Physics2Behavior/physics2runtimebehavior.ts

```diff
@@ -241,6 +246,8 @@ namespace gdjs {
     layers: any;
     masks: any;
     shapeScale: number = 1;
+    contactsStartedBetweenFrames: Array<any>;
+    contactsEndedBetweenFrames: Array<any>;
```

Comment:
Because there is nothing really "between frames" (something is always computed as part of the rendering of a frame), I would rather name these:

```suggestion
    contactsStartedThisFrame: Array<any>;
    contactsEndedThisFrame: Array<any>;
```

or 

```suggestion
    contactsJustStarted: Array<any>;
    contactsJustEnded: Array<any>;
```

Indeed, the simulation of the physics engine is done just before the events (in the do pre events). So the contacts are really detected as starting/ending when simulating *this* frame?

Wdyt @D8H @AlexandreSi ?